ee/modules/600-flant-integration/templates/pricing/daemonset.yaml

```diff
@@ -62,7 +62,7 @@ spec:
       - name: deckhouse-registry
       serviceAccountName: pricing
       {{- include "helm_lib_node_selector" (tuple . "master") | nindent 6 }}
-      {{- include "helm_lib_tolerations" (tuple . "master") | nindent 6 }}
+      {{- include "helm_lib_tolerations" (tuple . "any-node") | nindent 6 }}
```

Comment:
Why?

Reply:
Because we do not have toleration master now. It was similar to any node.